lib/vdsm/storage/lvmdevices.py

```diff
@@ -58,3 +63,38 @@ def enable():
     with lvmconf.LVMConfig() as config:
         config.setint("devices", "use_devicesfile", 1)
         config.save()
+
+
+def import_vg_devices(vgs, config=None):
+    """
+    Import devices of provided VGs into LVM devices file.
+    """
+    for vg in vgs:
+        run_vgimportdevices(vg, config)
+
+
+def run_vgimportdevices(vg, config=None):
+    """
+    Import underlying devices of provided VG into LVM devices file. Import is
+    done using vgimportdevices command. vgimportdevices takes into account
+    existing lvm filter, so if some devices are excluded by the filter, such
+    devices won't be imported. If all devices should be visible, function can
+    be called with config='devices { filter = ["a|.*|"] }'
```

Comment:
I wonder if we should use filter allowing all devices, or filter allowing all devices except devices used by ovirt vgs (based on the ovirt vg tag).

Would be useful to ask David about it.